chart/assets/operations/instance_groups/scheduler.yaml

```diff
@@ -98,6 +98,14 @@
       fieldRef:
         fieldPath: status.podIP
 
+- type: replace
+  path: /instance_groups/name=scheduler/jobs/name=loggr-syslog-binding-cache/properties?/quarks
+  value:
+    ports:
+    - name: binding-cache
+      protocol: TCP
+      internal: 9000
```

Comment:
This change is fixing the syslog drain CATs. I wonder how those tests passed in diego.